mgmtsystem_manual/models/mgmtsystem_manual.py

```diff
@@ -6,4 +6,5 @@ class MgmtSystemManual(models.Model):
 
     _inherit = 'mgmtsystem.system'
     _description = 'Manual'
-    manual = fields.Many2one('document.page', string='Manual')
+    manual_id = fields.Many2one('document.page', string='Manual')
+# old name of manual_id was manual
```

Comment:
shouldn't you use `oldname='manual'` to help with migration?
The comment line is unnecessary

Reply:
Agree